src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        int figureNumber = 10;
+
+        int figuresNumberInSetOne = 0;
+
+        if (figureNumber % 2 == 0) {
+            figuresNumberInSetOne = figureNumber / 2;
+        } else {
+            figuresNumberInSetOne = figureNumber / 2 + 1;
+        }
+
+        Figure [] result = new Figure[figureNumber];
+
+        for (int i = 0; i < figuresNumberInSetOne; i++) {
+            FigureSupplier figureSupplier = new FigureSupplier();

```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. Let's do it only once - before the loop starts.